juriscraper/pacer/http.py

```diff
@@ -279,8 +279,14 @@ def login(self, url=None):
             timeout=60,
             data=json.dumps(data),
         )
+        # Continue with login when response code is "200: OK"
+        if login_post_r.status_code == requests.codes.ok:
+            response_json = login_post_r.json()
+        else:
+            message = f"Unable connect to PACER site: '{login_post_r.status_code}: {login_post_r.reason}'"
+            logger.warning(message)
+            raise PacerLoginException(message)
```

Comment:
I think it's a little tidier to just have an early abort if statement here, but this is great otherwise:

```suggestion

        if login_post_r.status_code != requests.codes.ok:
            message = f"Unable connect to PACER site: '{login_post_r.status_code}: {login_post_r.reason}'"
            logger.warning(message)
            raise PacerLoginException(message)
            
        # Continue with login when response code is "200: OK"            
        response_json = login_post_r.json()            
```